samples/MessageModels/Commands/WriteCommand.json

```diff
@@ -0,0 +1,29 @@
+{
+  "id": "7cd70b85-2939-480d-917f-5d62c7f6b1b2",
+  "namespace": "models.messages.opcua.commandrequest",
+  "$schema": "https://iotpebsdmstorage.blob.core.windows.net/schemas/message_schemas/2020-03-31/message.opcua.command.request.schema.json",
+  "metadata": {
+    "timestamp": "2020-02-17T09:50:29+00:00",
+    "correlationIds": [],
+    "senderIdentifier": {
+      "id": "7cd70b85-2939-480d-917f-5d62c7f6b1b2",
+      "name": "IPS-T",
+      "type": "application",
+      "route": "/control"
+    },
+    "destinationIdentifiers": [
+    ]
+  },
+  "payload": {
+    "requestTarget": {
+      "endpointUrl": "opc.tcp://opcuaserverintegration:52210/"
+    },
+    "requests": [
+      {
+        "commandType": "write",
+        "nodeId": "ns=2;i=16397",
+        "value": $WRITEVALUE$
```

Comment:
This is a placeholder that we used to replace in integration tests.
We should replace this with an actual string so that it is not confusing what to do or put here.